Move imported packages to a proper path

Fix path for packages. Move them to user.

When using `go get` the repositoy will get cloned inside `$GOPATH/src/github.com/$USER/$REPO`.

When I do a `go install` in the path `go/src/github.com/Tinker-Ware/gh-service` i get the following error:

```
main.go:10:2: cannot find package "github.com/gh-service/infraestructure" in any of:
	/usr/local/go/src/github.com/gh-service/infraestructure (from $GOROOT)
	/home/vagrant/go/src/github.com/gh-service/infraestructure (from $GOPATH)
main.go:11:2: cannot find package "github.com/gh-service/interfaces" in any of:
	/usr/local/go/src/github.com/gh-service/interfaces (from $GOROOT)
	/home/vagrant/go/src/github.com/gh-service/interfaces (from $GOPATH)
main.go:12:2: cannot find package "github.com/gh-service/usecases" in any of:
	/usr/local/go/src/github.com/gh-service/usecases (from $GOROOT)
	/home/vagrant/go/src/github.com/gh-service/usecases (from $GOPATH)
vagrant@tinkerware:~/go/src/github.com/Tinker-Ware/gh-service$ go build
main.go:10:2: cannot find package "github.com/gh-service/infraestructure" in any of:
	/usr/local/go/src/github.com/gh-service/infraestructure (from $GOROOT)
	/home/vagrant/go/src/github.com/gh-service/infraestructure (from $GOPATH)
main.go:11:2: cannot find package "github.com/gh-service/interfaces" in any of:
	/usr/local/go/src/github.com/gh-service/interfaces (from $GOROOT)
	/home/vagrant/go/src/github.com/gh-service/interfaces (from $GOPATH)
main.go:12:2: cannot find package "github.com/gh-service/usecases" in any of:
	/usr/local/go/src/github.com/gh-service/usecases (from $GOROOT)
	/home/vagrant/go/src/github.com/gh-service/usecases (from $GOPATH)
```

main.go is looking for the packages in a diferent path (without the user). Please move them to a path with the user included.

--- a/infrastructure/infraestructure_suite_test.go
+++ b/infrastructure/infraestructure_suite_test.go
@@ -1,4 +1,4 @@
-package infraestructure_test
+package infrastructure_test
 
 import (
 	. "github.com/onsi/ginkgo"
@@ -7,7 +7,7 @@ import (
 	"testing"
 )
 
-func TestInfraestructure(t *testing.T) {
+func Testinfrastructure(t *testing.T) {
 	RegisterFailHandler(Fail)
-	RunSpecs(t, "Infraestructure Suite")
+	RunSpecs(t, "infrastructure Suite")
 }

--- a/infrastructure/infraestructure_test.go
+++ b/infrastructure/infraestructure_test.go
@@ -1,16 +1,16 @@
-package infraestructure_test
+package infrastructure_test
 
 import (
 	"io/ioutil"
 	"os"
 
-	. "github.com/gh-service/infraestructure"
+	. "github.com/gh-service/infrastructure"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 )
 
-var _ = Describe("Infraestructure", func() {
+var _ = Describe("infrastructure", func() {
 
 	Describe("Read a YAML", func() {
 		Context("Create a YAML on the fly", func() {

--- a/infrastructure/infraestructure_test.go
+++ b/infrastructure/infraestructure_test.go
@@ -4,7 +4,7 @@ import (
 	"io/ioutil"
 	"os"
 
-	. "github.com/gh-service/infrastructure"
+	. "github.com/Tinker-Ware/gh-service/infrastructure"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"

--- a/infrastructure/logger.go
+++ b/infrastructure/logger.go
@@ -1,1 +1,1 @@
-package infraestructure
+package infrastructure

--- a/infrastructure/yaml.go
+++ b/infrastructure/yaml.go
@@ -1,4 +1,4 @@
-package infraestructure
+package infrastructure
 
 import (
 	"io/ioutil"

--- a/interfaces/adapter.go
+++ b/interfaces/adapter.go
@@ -5,7 +5,7 @@ import (
 	"net/http"
 	"time"
 
-	"github.com/gh-service/domain"
+	"github.com/Tinker-Ware/gh-service/domain"
 )
 
 type Adapter func(http.Handler) http.Handler

--- a/interfaces/github.go
+++ b/interfaces/github.go
@@ -7,7 +7,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/gh-service/domain"
+	"github.com/Tinker-Ware/gh-service/domain"
 	"github.com/google/go-github/github"
 	"golang.org/x/oauth2"
 

--- a/interfaces/github_repo_test.go
+++ b/interfaces/github_repo_test.go
@@ -112,7 +112,7 @@ var _ = Describe("GithubRepo", func() {
 			Author:  "iasstest",
 			Message: "Hello",
 			Branch:  "master",
-			Email:   "infraestructuretest@gmail.com",
+			Email:   "infrastructuretest@gmail.com",
 		}
 		It("Should create files in the repo", func() {
 			_, err := repo.CreateRepo(username, reponame, "", false)

--- a/interfaces/github_repo_test.go
+++ b/interfaces/github_repo_test.go
@@ -8,8 +8,8 @@ import (
 
 	"net/url"
 
-	"github.com/gh-service/domain"
-	. "github.com/gh-service/interfaces"
+	"github.com/Tinker-Ware/gh-service/domain"
+	. "github.com/Tinker-Ware/gh-service/interfaces"
 	"github.com/google/go-github/github"
 	"golang.org/x/oauth2"
 

--- a/interfaces/webservice.go
+++ b/interfaces/webservice.go
@@ -6,7 +6,7 @@ import (
 	"net/http"
 	"strconv"
 
-	"github.com/gh-service/domain"
+	"github.com/Tinker-Ware/gh-service/domain"
 	"github.com/gorilla/mux"
 	"github.com/gorilla/sessions"
 	"golang.org/x/oauth2"

--- a/main.go
+++ b/main.go
@@ -6,10 +6,10 @@ import (
 	"fmt"
 	"net/http"
 
+	"github.com/Tinker-Ware/gh-service/infrastructure"
+	"github.com/Tinker-Ware/gh-service/interfaces"
+	"github.com/Tinker-Ware/gh-service/usecases"
 	"github.com/codegangsta/negroni"
-	"github.com/gh-service/infrastructure"
-	"github.com/gh-service/interfaces"
-	"github.com/gh-service/usecases"
 	"github.com/gorilla/mux"
 	"github.com/gorilla/sessions"
 )

--- a/main.go
+++ b/main.go
@@ -7,7 +7,7 @@ import (
 	"net/http"
 
 	"github.com/codegangsta/negroni"
-	"github.com/gh-service/infraestructure"
+	"github.com/gh-service/infrastructure"
 	"github.com/gh-service/interfaces"
 	"github.com/gh-service/usecases"
 	"github.com/gorilla/mux"
@@ -24,7 +24,7 @@ func main() {
 
 	flag.Parse()
 
-	config, err := infraestructure.GetConfiguration(*confFilePath)
+	config, err := infrastructure.GetConfiguration(*confFilePath)
 	if err != nil {
 		fmt.Println(err.Error())
 		panic("Cannot parse configuration")

--- a/usecases/files.go
+++ b/usecases/files.go
@@ -12,7 +12,7 @@ func (interactor GHInteractor) CreateFile(file domain.File, author domain.Author
 }
 
 func (interactor GHInteractor) AddFiles(files []domain.File, author domain.Author, username, repo string) error {
-	err := interactor.AddFiles(files, author, username, repo)
+	err := interactor.GithubRepository.AddFiles(files, author, username, repo)
 	if err != nil {
 		return err
 	}

--- a/usecases/files.go
+++ b/usecases/files.go
@@ -1,6 +1,6 @@
 package usecases
 
-import "github.com/gh-service/domain"
+import "github.com/Tinker-Ware/gh-service/domain"
 
 func (interactor GHInteractor) CreateFile(file domain.File, author domain.Author, username, repo string) error {
 

--- a/usecases/ghoauth.go
+++ b/usecases/ghoauth.go
@@ -1,6 +1,6 @@
 package usecases
 
-import "github.com/gh-service/domain"
+import "github.com/Tinker-Ware/gh-service/domain"
 
 type GHInteractor struct {
 	GithubRepository GithubRepository

--- a/usecases/keys.go
+++ b/usecases/keys.go
@@ -1,6 +1,6 @@
 package usecases
 
-import "github.com/gh-service/domain"
+import "github.com/Tinker-Ware/gh-service/domain"
 
 func (interactor GHInteractor) ShowKeys(username string) ([]domain.Key, error) {
 

--- a/usecases/repos.go
+++ b/usecases/repos.go
@@ -1,6 +1,6 @@
 package usecases
 
-import "github.com/gh-service/domain"
+import "github.com/Tinker-Ware/gh-service/domain"
 
 func (interactor GHInteractor) ShowRepos(username string) ([]domain.Repository, error) {
 

--- a/usecases/usecases_test.go
+++ b/usecases/usecases_test.go
@@ -60,7 +60,7 @@ var _ = Describe("Usecases", func() {
 				repos, err := interactor.ShowRepos(username)
 				Ω(err).ShouldNot(HaveOccurred())
 
-				Ω(repos).Should(HaveLen(1))
+				Ω(len(repos)).Should(BeNumerically(">", 0))
 			})
 
 		})

--- a/usecases/usecases_test.go
+++ b/usecases/usecases_test.go
@@ -50,7 +50,7 @@ var _ = Describe("Usecases", func() {
 		Context("Test repo functionality", func() {
 
 			It("Should create a repo with the test account", func() {
-				repo, err := interactor.CreateRepo(username, token, reponame, "", false)
+				repo, err := interactor.CreateRepo(username, reponame, "", false)
 
 				Ω(err).ShouldNot(HaveOccurred())
 				Ω(*repo.FullName).Should(ContainSubstring(reponame))
@@ -72,7 +72,7 @@ var _ = Describe("Usecases", func() {
 			}
 
 			It("Should Create a Key", func() {
-				err := interactor.CreateKey(username, token, &key)
+				err := interactor.CreateKey(username, &key)
 				Ω(err).ShouldNot(HaveOccurred())
 
 				Ω(*key.ID).ShouldNot(BeZero())
@@ -129,7 +129,7 @@ var _ = Describe("Usecases", func() {
 				Email:   "infrastructuretest@gmail.com",
 			}
 			It("Should create files in the repo", func() {
-				_, err := interactor.CreateRepo(username, token, repo, "", false)
+				_, err := interactor.CreateRepo(username, repo, "", false)
 				Ω(err).ShouldNot(HaveOccurred())
 
 				err = interactor.AddFiles(files, author, username, repo)

--- a/usecases/usecases_test.go
+++ b/usecases/usecases_test.go
@@ -101,7 +101,7 @@ var _ = Describe("Usecases", func() {
 				Author:  "iasstest",
 				Message: "Add file",
 				Branch:  "master",
-				Email:   "infraestructuretest@gmail.com",
+				Email:   "infrastructuretest@gmail.com",
 			}
 
 			It("Should create a file in the repo", func() {
@@ -126,7 +126,7 @@ var _ = Describe("Usecases", func() {
 				Author:  "iasstest",
 				Message: "Hello",
 				Branch:  "master",
-				Email:   "infraestructuretest@gmail.com",
+				Email:   "infrastructuretest@gmail.com",
 			}
 			It("Should create files in the repo", func() {
 				_, err := interactor.CreateRepo(username, token, repo, "", false)

--- a/usecases/usecases_test.go
+++ b/usecases/usecases_test.go
@@ -12,9 +12,9 @@ import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 
-	"github.com/gh-service/domain"
-	"github.com/gh-service/interfaces"
-	. "github.com/gh-service/usecases"
+	"github.com/Tinker-Ware/gh-service/domain"
+	"github.com/Tinker-Ware/gh-service/interfaces"
+	. "github.com/Tinker-Ware/gh-service/usecases"
 )
 
 var _ = Describe("Usecases", func() {

